bundles/org.openhab.binding.emotiva/pom.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>4.2.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.emotiva</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: Emotiva Binding</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.assertj</groupId>
```

Comment:
Is this dependency essential? As i have seen several bindings with unittests without any dependency.

Reply:
I guess its mostly a matter of taste. The whole test suite use the fluent based assertThat() instead of the normal assertEquals/True/False.

Reply:
Not entirely taste. We try prevent adding unnecessary dependency’s and keep the dependency footprint as small as reasonable possible.

Reply:
I saw that the fineoffsetweatherstation binding was using it, so I though that it would be fine.

Reply:
Missed that one, in that case let's continue.

Reply:
@espenaf - constaint-based assert model is nice, but perhaps you can use the org.hamcrest matchers instead like many other bindings? See for example: https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.energidataservice/src/test/java/org/openhab/binding/energidataservice/internal/action/EnergiDataServiceActionsTest.java

Reply:
Hehe. One include of assertj might be a mistake, adding one more quickly creates a precedence, so I just rewrote the tests and removed assertj from this binding.